src/routes/InventoryTable.js

```diff
@@ -51,6 +51,7 @@ import {
 } from '../components/InventoryTable/ActionWithRBAC';
 import uniq from 'lodash/uniq';
 import useInsightsNavigate from '@redhat-cloud-services/frontend-components-utilities/useInsightsNavigate/useInsightsNavigate';
+import { difference, map } from 'lodash';
```

Comment:
IIRC, with this import we will have the issue of loading the whole lodash library. And since it's not yet solved, we better rewrite it to individual imports per each function required.

```suggestion
import difference from 'lodash/difference';
```

Reply:
Good point :smile: